streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -1883,3 +1886,20 @@ public static void main(final String[] args) {
         System.out.println(CONFIG.toHtml(4, config -> "streamsconfigs_" + config));
     }
 }
+
+
+    public Map<String, Object> getMainConsumerConfigs(...) {
```

Comment:
`StreamsConfig` is public API and we cannot just modify it w/o a KIP. -- Also, why do we need this new method to begin with? We already have `getMainConsumerConfigs(...)`.